packages/framework-provider-azure-infrastructure/src/infrastructure/synth/terraform-web-pubsub.ts

```diff
@@ -0,0 +1,24 @@
+import { TerraformStack } from 'cdktf'
+import { toTerraformName } from '../helper/utils'
+import { resourceGroup, webPubsub } from '@cdktf/provider-azurerm'
+import { AzurermProvider } from '@cdktf/provider-azurerm/lib/provider'
+
+export class TerraformWebPubsub {
+  static build(
+    providerResource: AzurermProvider,
+    terraformStackResource: TerraformStack,
+    resourceGroupResource: resourceGroup.ResourceGroup,
+    appPrefix: string
+  ): webPubsub.WebPubsub {
+    const id = toTerraformName(appPrefix, 'wps')
+
+    return new webPubsub.WebPubsub(terraformStackResource, id, {
+      name: `${resourceGroupResource.name}wps`,
+      location: resourceGroupResource.location,
+      resourceGroupName: resourceGroupResource.name,
+      sku: 'Free_F1',
+      capacity: 1,
```

Comment:
Is there any important limitation with this consumption SKU and capacity?

Reply:
The sku options are: `Free_F1` and `Standard_S1` and the capacity is `the number of units associated with this Web Pubsub resource` but the Free sku only accept 1 unit. The limitations are:

<img width="1325" alt="image" src="https://user-images.githubusercontent.com/111118818/231728297-5e10ae20-8d03-42ac-9d59-17a63566164b.png">

Reply:
@gonzalojaubert I think this should go into the documentation and we should create an issue to allow configuring this in the config